### PR TITLE
[Feature] Added CRUD in Tutorial Model #217

### LINKED ISSUE #207: [Feat]: [Adding CURD to Tutorial]
## Description

Add CURD to tutorial.js

## Proposed Solution

Use any model with CURD as reference

### PR DESCRIPTION
**Pull request by Aryanzs**
## Description
Added CRUD operations to the tutorial model

## Fixes
Fixes #207 

## Checklist

- [x] Code follows project's style guidelines.
- [x] Changes are documented appropriately.

### CODE REVIEW

File: models/tutorial.js [hunk 1]
```diff
@@ -13,3 +13,39 @@ const tutorialSchema = {
...
+
+// Update tutorials based on a given filter and update data
+async function update(filter, updateObject, options = { multi: true }) {
+  const updateResult = await Tutorial.updateMany(filter, { $set: updateObject }, options);
+  return updateResult.acknowledged;
+}
+
+// Delete tutorials based on a given filter
+async function remove(filter) {
+  const deleteResult = await Tutorial.deleteMany(filter).exec();
```
Comment (Aastha-S-Rai) [inline models/tutorial.js:40]: Why is there exec() function used here? I don't think we need it.

Comment (Aryanzs) [reply models/tutorial.js:40]: Thinking of it as confirming your request to delete items from the database. Once you press "Go" (by using .exec()). Should i change it and make another commit or let it be?

Comment (TejasNair9977) [reply models/tutorial.js:40]: it should be fine

**Review by TejasNair9977 — APPROVED** [review]
LGTM

**Pull request merged**